LondonBritishLibrary/orient/BLorient9036.xml

```diff
@@ -0,0 +1,297 @@
+        <fileDesc>
+            <titleStmt>
+                <title>Psalter</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 9036</idno>
+                        <altIdentifier><idno>Strelcyn 22</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="134vb"/>
+                            <title ref="LIT2701Dawit"/>
+                        <msItem xml:id="ms_i1.1">
+                            <locus from="3r" to="101r"/>
+                            <title ref="LIT2000Mazmur"/>
+                            <note>The titles of each of the psalms are the new ones.</note>
+                            <incipit xml:lang="gez">ነዓ፡ ኀቤየ፡ ኦዳዊት፡ ንጉሠ፡ እ፳ል። 
+                                በዓለ፡ መዝሙር፡ ሠናይ፡ ወጥ<supplied reason="undefined">ዑ</supplied>ም፡ ቃል። 
+                                ታለብወኒ፡ ነገር፡ ወፍካሬ፡ ኵሉ፡ አምሳል።
+                                ከመ፡ ዕወድሳ፡ ለማርያም፡ ድንግል።
+                                እንዘ፡ እጻርሕ፡ ወእብል።
+                                ተግሣጽ፡ ለኵሉ።
+                                ፍካሬ፡ ዘጻድቃን፡ ወዘኃአጥንን።
+                                <title>ብፁዕ፡ ብእሲ፡</title></incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.2">
+                            <locus from="101r" to="118r"/>
+                            <title ref="LIT1233Cantic"/>
+                            <msItem xml:id="ms_i1.2.1">
+                                <locus from="101r" to="111v"/>
+                                <title ref="LIT1828Mahale"/>
+                                <note>The title of each canticle is preceded by the doxology: <q xml:lang="gez">ስብሐት፡ ለአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ።</q></note>
+                                <msItem xml:id="ms_i1.2.1.1">
+                                    <locus from="101r" to="102r"/>
+                                    <title ref="LIT2277salotu"/>
```

Comment:
NB actually when a text is a part/section of a larger text, then we refer to it with a sub-id if it is not an independent circulation, that is `LIT1828Mahale#Moses1` instead of `LIT2277salotu `etcetera (also in other Psalters or other similar cases). use `LIT2277salotu `when it appears outside of `LIT1828Mahale`